pakyow-core/lib/core/middleware/req_path_normalizer.rb

```diff
@@ -0,0 +1,27 @@
+require 'core/call_context'
+
+module Pakyow
```

Comment:
I don't see where this is being setup, but it looks like you're telling `Rack::Builder` to use an instance rather than the class.

```ruby
# do this
builder.use Pakyow::Middleware::ReqPathNormalizer

# rather than this
builder.use Pakyow::Middleware::ReqPathNormalizer.new
```